include/seqan/blast.h

```diff
@@ -39,7 +39,7 @@
 
 #include <seqan/platform.h>
 
-#ifndef PLATFORM_WINDOWS_VS
+#ifndef STD_LIB_VS
```

Comment:
this is actually `ndef COMPILER_MSVC` but please try out, if this might not be fixed with the newest MSVC, I think this could be removed now... thanks!